helpers/integration.go

```diff
@@ -554,3 +555,11 @@ func IsObjectYoungerThanThreshold(obj metav1.Object, threshold time.Duration) bo
 
 	return durationSinceObjectCreation < threshold
 }
+
+func IsPipelineRunInFinalState(detail *integrationteststatus.IntegrationTestStatusDetail) bool {
```

Comment:
Thanks for adding this function, can you also add some basic unit test coverage for it just in case?

Reply:
totally unittest

Reply:
technically it could be method of `IntegrationTestStatus`

```golang
func (s IntegrationTestStatus) IsFinal() bool {
	switch s {
	case
		IntegrationTestStatusDeleted,
		IntegrationTestStatusEnvironmentProvisionError_Deprecated,
		IntegrationTestStatusDeploymentError_Deprecated,
		IntegrationTestStatusTestFail,
		IntegrationTestStatusTestPassed,
		IntegrationTestStatusTestInvalid:
		return true
	}
	return false
}
```

```go
if testDetails.Status.IsFinal() {
   # do something
}
```

Reply:
Function was moved to belong to ITS and is covered by unit tests [here](https://github.com/redhat-appstudio/integration-service/pull/626/files#diff-ef200ddee2492694a51e155d9bbcc48f03a1d67408a896b7d00daa46c5802342R72)